BackofficeBundle/Resources/translations/messages.fr.yml

```diff
@@ -505,6 +505,7 @@ open_orchestra_backoffice:
         version:
             create: Nouvelle version
             display: version
+            updated: modifiée le
```

Comment:
@KevinArtus: Your labels are not rigorously identical in english and in french: the translation is not accurate and the uppercase is not respected. I suggest something like ``Dernière modification`` for the french translation or ``Updated on`` for the english one